src/api/entities.js

```diff
@@ -1,25 +1,24 @@
-import { TOPOLOGICAL_INVENTORY_API_BASE } from '../Utilities/Constants';
-import { sourcesViewDefinition } from '../views/sourcesViewDefinition';
+import axios from 'axios';
+import { DefaultApi as SourcesDefaultApi } from '@redhat-cloud-services/sources-client';
 import find from 'lodash/find';
 
-/*global require*/
-let TopologicalInventory = require('@manageiq/topological_inventory');
+import { sourcesViewDefinition } from '../views/sourcesViewDefinition';
+import { SOURCES_API_BASE } from '../Utilities/Constants';
 
+const axiosInstance = axios.create();
 let apiInstance;
 
-export function getApiInstance() {
+export function getSourcesApi() {
     if (apiInstance) {
         return apiInstance;
     }
 
-    apiInstance = new TopologicalInventory.DefaultApi();
-    let defaultClient = TopologicalInventory.ApiClient.instance;
-    defaultClient.basePath = TOPOLOGICAL_INVENTORY_API_BASE;
+    apiInstance = new SourcesDefaultApi(undefined, SOURCES_API_BASE, axiosInstance);
```

Comment:
You might want to use response interceptor to return data instead of object with data
```
instance.interceptors.response.use((response) => {
  return response.data;
});
```